app/server/tests/api-register.test.js

```diff
@@ -25,33 +27,33 @@ describe('Testing optimal inputs for register', () => {
     });
 });
 
-describe('Testing sub-optimal inputs for register - all return 400', () => {
+describe('Testing sub-optimal inputs for register', () => {
     afterAll(async () => {
         await users.deleteUsers();
     });
 
     test('api rejects registration if password doesn\'t meet character specifications', async () => {
         let response = await users.register(name.gen(), email.gen(), 'badpasswo1!');
-        expect(response.status).toBe(400);
+        expect(response.status).not.toBe(201);
```

Comment:
Why are we changing these to look for not 201? What happens if this returned a 503?

Reply:
Or better yet, a 418.